assets/app/views/_project-nav.html

```diff
@@ -1,42 +1,36 @@
-<div class="navbar-project">
-  <div class="navbar-project-menu">
-    <div class="form-group">
-      <label class="control-label" for="boostrapSelect">Projects</label>
-      <!-- TODO add multiple attribute back to the select tag when we support selecting multiple -->
-      <select class="selectpicker form-control" data-selected-text-format="count>3" id="boostrapSelect"></select>
+<div class="navbar-project container-fluid">
+  <div class="row">
+    <div class="col-sm-4">
+      <div class="form-group">
+        <label class="control-label" for="boostrapSelect">Projects</label>
+        <!-- TODO add multiple attribute back to the select tag when we support selecting multiple -->
+        <select class="selectpicker form-control" data-selected-text-format="count>3" id="boostrapSelect"></select>
+      </div>
     </div>
-  </div>
```

Comment:
@liggit Looks OK to me.

![openshift_management_console](https://cloud.githubusercontent.com/assets/1167259/7536828/4400239c-f561-11e4-9500-c6551efd6551.png)

![openshift_management_console_and_wrap_navbar_controls_naturally_for_small_screens_by_spadgett_ _pull_request__2098_ _openshift_origin_and_1__hack_serve-local-assets_sh__bash_](https://cloud.githubusercontent.com/assets/1167259/7536831/59ba2ea8-f561-11e4-8bdb-439e89d21ba3.png)